layouts/partials/comments/provider/cactus.html

```diff
@@ -0,0 +1,27 @@
+{{- with .Site.Params.comments.cactus -}}
+<style>
+    .cactus-editor-textarea {
+        color: var(--body-text-color);
+    }
+
+    .cactus-comment-header {
+        color: var(--card-text-color-main);
+    }
+
+    .cactus-message-text>p {
+        color: var(--body-text-color);
+    }
+</style>
+
+<div id="comment-section"></div>
+
+<script>
+    initComments({
+        node: document.getElementById("comment-section"),
+        defaultHomeserverUrl: "{{ .defaultHomeserverUrl | safeHTML }}",
```

Comment:
It should be ` | safeJS` I think

Reply:
Will test that :)

Reply:
Sorry that I only now respond. This seems to do the same in this case but I will change it :)